vnext/Microsoft.ReactNative/ReactPointerEventArgs.idl

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "DocString.h"
+#include "NamespaceRedirect.h"
+
+namespace Microsoft.ReactNative {
+  [experimental]
+  [webhosthidden]
+  DOC_STRING("Pointer event type enum.")
```

Comment:
I think instead of this doc string we should have individual doc strings for the enums that explain when they are used (if that adds value). If these mimic XAML or other WinRT enums we should point to those (you can reference them with the syntax `@TypeName.Property`)